book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals.push(rental)
+    rental.book = self
+  end
```

Comment:
- [ ] Please the `book` itself should be sent as a parameter to create the new rental achieving the 'has-many' association since the `Rental` class' initialize method has handles adding the rental to the `rentals` array in the `Book` class.
See HINT below 👇 
```ruby
def add_rental(person, date)
  Rental.new(date, person, self)
end
```